### PR TITLE
chore: rtl locale selector popup

### PR DESCRIPTION
**Pull request by DasMergo**
Screenshot 2024-01-04 130739
rtl for locale selector popup

### CODE REVIEW

File: packages/payload/src/admin/components/elements/Header/index.scss [hunk 1]
```diff
@@ -15,8 +15,7 @@
 [dir='rtl'] &__localizer {
     right: unset;
     left:  base(4.5)
-  }
-  
+
```
Comment (DanRibbens) [inline packages/payload/src/admin/components/elements/Header/index.scss:18]: It looks like this was removed by accident maybe?
```
ERROR in ./packages/payload/src/admin/components/elements/Header/index.scss (./node_modules/.pnpm/css-loader@5.2.7_webpack@5.88.2/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[2].use[1]!./node_modules/.pnpm/postcss-loader@6.2.1_postcss@8.4.31_webpack@5.88.2/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[2].use[2]!./node_modules/.pnpm/sass-loader@12.6.0_sass@1.69.4_webpack@5.88.2/node_modules/sass-loader/dist/cjs.js!./packages/payload/src/admin/components/elements/Header/index.scss)
Module build failed (from ./node_modules/.pnpm/sass-loader@12.6.0_sass@1.69.4_webpack@5.88.2/node_modules/sass-loader/dist/cjs.js):
SassError: expected "}".
    ╷
192 │ }
    │  ^
    ╵
  packages/payload/src/admin/components/elements/Header/index.scss 192:2  root stylesheet
```

**Review by DanRibbens — CHANGES_REQUESTED** [review]
Thanks for maintaining the RTL support for the admin UI. Can you fix the header scss?

**Review by DasMergo — COMMENTED** [review]
fixed

**Review by DanRibbens — APPROVED** [review]
Thanks!

**Pull request merged**